src/components/App.js

```diff
@@ -34,14 +35,19 @@ export default (props) => {
   const publicURL = process.env.PUBLIC_URL ? process.env.PUBLIC_URL : '/'
 
   return (
-    <GlobalCSS>
-      <BrowserRouter basename={publicURL}>
-        <Switch>
-          <Route path="/embed" component={Embed} />
-          <Route path="/" component={Election} />
-          <Redirect to="/" />
-        </Switch>
-      </BrowserRouter>
-    </GlobalCSS>
+    <GoogleReCaptchaProvider
+      reCaptchaKey="6LfqhYUkAAAAAAp-KRpwLPTROD1jrAQlQQV_L8R8"
```

Comment:
```suggestion
      reCaptchaKey="process.env.GOOGLE_RECAPTCHA_KEY"
```

Нека преместим този ключ в `.env`?
`.env` файлът се използва само за development и се използват други стойности при deployment, които ще ги наглася допълнително.

Reply:
Ще го изместя, благодаря.